File: chanukah.rb
p=gets.to_i;
for i in 0...p
  n,k=gets.chomp.split.map(&:to_i)
  count=0;
  for j in 1..k
      count = count +j;
  end
  print n," ", count+k,"\n"
end
    

File: chanukah.py
p=int(input())
for i in range(0,p):
  lst = list(input().split())
  n=int(lst[0])
  k=int(lst[1])
  count=0;
  for j in range(1,k+1):
      count = count +j;
  print (n,count+k,sep=" ",end="\n")

